pkg/trace/sampler/scoresampler.go

```diff
@@ -70,11 +59,9 @@ func (s *ScoreSampler) Sample(now time.Time, trace pb.Trace, root *pb.Span, env
 
 	rate := s.getSignatureSampleRate(signature)
 
-	return s.applySampleRate(root, rate)
-}
+	sampled := s.applySampleRate(root, rate)
 
-func (e *ErrorsSampler) UpdateTargetTPS(targetTPS float64) {
-	e.ScoreSampler.Sampler.updateTargetTPS(targetTPS)
+	return sampled
```

Comment:
Any reason to pull this out to a variable instead of just returning `s.applySampleRate` as before?